src/Sulu/Bundle/AdminBundle/Tests/Features/change_language.feature

```diff
@@ -8,5 +8,12 @@ Feature: Change the language of the backoffice
 
     Scenario: Change the language
         Given I am on "/admin"
+        And I wait a second
```

Comment:
Is that really necessary? Isn't there an event you can wait for? In general waiting is a bad idea, since it makes the tests flaky. Same for the next wait.